Context/NikeDbContext.cs

```diff
@@ -91,8 +91,6 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
            
             modelBuilder.Entity<FlockType>().Property(u => u.Name).IsRequired();
 
-            modelBuilder.Entity<FlockType>().Property(u => u.Description).IsRequired();
```

Comment:
Why did remove this here?

Reply:
You**

Reply:
You asked me too make d flocktype description optional